### DISCUSSION TITLE
Alexa, integrazione haaska - non aggiorna nomi dispositivi + errore

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2268] [GENERALE - Discussioni generiche]
Ciao a tutti, 
ho integrato Alexa con haaska, tutto ok, test ok, dispositivi importati nella skill.
sto ora cambiando i nomi di alcuni dispositivi in HA ma vedo che non vengono aggiornati in alexa.
altro problema che ho notato è questo errore nel core :
```
Error when sending ChangeReport for light.luce_segnapassi_lightbulb to Alexa: INVALID_ACCESS_TOKEN_EXCEPTION: Access token is not valid.
```
premetto che alexa/HA risponde ai comandi regolarmente...

**Comment by snakuzzo** [comment]
https://community.home-assistant.io/t/homeassistant-components-alexa-state-report-error-when-sending-changereport-to-alexa-invalid-access-token-exception-access-token-is-not-valid/214853/11

**Reply by Matty1184** [reply]
grazie per il link, ho attivato la funzione 'Send Alexa Events' che non avevo abilitato.
purtroppo però non mi è cambiato nulla, anzi ora mi compare anche questo errore:
```
Error calling LWA to get auth token - Amazon Alexa
```

**Reply by snakuzzo** [reply]
mi spiace, non so che dirti

**Reply by Matty1184** [reply]
trovato il problema!
lo descrivo qua qualora possa capitare una svista come è successo a me!
nel configuration.yaml avevo inserito user e secret errati per il tipo di autenticazione (avevo inserito quelli di AWS generati nei primi step della creazione dell'account)
quelli da prendere in considerazione sono quelli nella pagina "Alexa Developer Console
" nella tab "Permission" e nella stessa pagina abilitare "Send Alexa Events".

**Reply by snakuzzo** [reply]
Quindi avevi sbagliato a seguire la guida :)

Chiudo
